src/generateRandomInteger/generateRandomInteger.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const generateRandomInteger = require('./generateRandomInteger');
+
+describe('Testing generateRandomInteger', () => {
+  test('Should return 1 for 1', () => {
+    expect(generateRandomInteger(1))
+      .toBe(1);
+  });
+
+  test('Should return integer', () => {
+    const number = generateRandomInteger(10);
+    expect(number % Math.trunc(number))
+      .toBe(0);
+  });
+
+  test('Should generate numbers in range [1, N] when called many times',
```

Comment:
Let's add test that min and max are not equal. Currently function can just return N.